rfc8446.xml

```diff
@@ -3961,7 +3964,7 @@ generating the client_early_traffic_secret.</t>
        Derive-Secret(., "derived", "")
              |
              v
-  0 -> HKDF-Extract = Master Secret
+   0 -> HKDF-Extract = Master Secret
```

Comment:
These two unify the left margin for the HKDF-Extract invocations in this figure.